packages/synapse-react-client/src/components/Markdown/MarkdownCollapse.tsx

```diff
@@ -27,8 +28,9 @@ export const MarkdownCollapse = (props: MarkdownCollapseProps) => {
   const [wordCount, setWordCount] = useState<number>()
   const [plainText, setPlainText] = useState<string>()
 
-  const onMarkdownProcessingDone = (textContent: string | null | undefined) => {
-    if (textContent) {
+  const onMarkdownProcessingDone = (htmlContent: string | null | undefined) => {
```

Comment:
onMarkdownProcessingDone now returns html content rather than the stripped html text (html used in other use case)